### ISSUE TITLE
Ability to "conda env create" from Anaconda cloud with --no-deps argument? 

### ISSUE THREAD

**Issue by ajoros** [conda/conda#2941]
I've created a an environment that I want to use on https://anaconda.org/ajoros/ves_env_win64
[pardon my newbieness] 

Is it possible to perform a command like:

`conda env create ajoros/ves_env_win64`

without it installing the dependencies of the packages in my environment.yaml. I am looking for `conda env create` to mimic the action that occurs when using `conda install --no-deps <package name>`. 

For example it installs`matplotlib=1.5.1=np111py35_0`, but it also installs all its dependencies like pyqt and qt when I dont want it to. I want it to install the base matplotlib package without any dependencies. I want to do this because later on I install @mmcauliffe's version of pyqt5 and qt5 from anaconda cloud (as you can see in the yaml file on Anaconda Cloud)

In short, I'm looking to run `conda env create ajoros/ves_env_win64` and create an environment of **ONLY** the packages listed in my yaml file, and nothing more ... (See YML file here:  http://pastebin.com/MXcnpdqh)

Thanks

aj

**Comment by mmcauliffe** [comment]
Just a heads up, PyQt5.6 is installable from pip now: http://pyqt.sourceforge.net/Docs/PyQt5/installation.html, so you might be able to avoid the conflicts that way.

**Comment by ajoros** [comment]
Thanks @mmcauliffe , but the issue is still that when I create an environment from my yaml file on Anaconda Cloud, it attempts to install matplotlib and ALL the conda dependencies such as pyqt and qt (which are still version 4 on conda official repositories)

**Comment by mmcauliffe** [comment]
Right, so my suggestion would be to remove my channel from there, delete the lines:

```
  - icu=56.1=0
  - pyqt5=5.5.1=py35_0
  - qt5=5.5.1=0
  - sip=4.17=py35_0
```

from the dependencies part, and then add the line:

```
  - pyqt5=5.6
```
to the pip part.

That shouldn't result in any dependency conflicts, PyQt4 and PyQt5 will both be installed, but that should be okay.  I ran just a quick test by doing:

```
conda create -n test python=3
source activate test
conda install matplotlib
pip install pyqt5
python
>>> import PyQt5
>>> import matplotlib
```
and everything seemed to be working okay, so it should work alright on the anaconda-cloud too (though I don't have any experience with it).

**Comment by ajoros** [comment]
OK I edited by environment.yml file (seen below) but it's returning this error:
```
Using Anaconda Cloud api site https://api.anaconda.org
Fetching package metadata .......
Solving package specifications: .
Error: Package missing in current win-64 channels:
  - pyqt5 5.6*
```

I dont think pyqt5 is available on the conda repository, and i'm not sure how to install via pip if i'm using `conda env create ajoros/ves_env_win64`

Here is my YML:

```
name: ves_env_win64
dependencies:
  - backports=1.0=py35_0
  - bzip2=1.0.6=vc14_3
  - cycler=0.10.0=py35_0
  - decorator=4.0.10=py35_0
  - freetype=2.5.5=vc14_1
  - get_terminal_size=1.0.0=py35_0
  - jpeg=8d=vc14_0
  - libpng=1.6.22=vc14_0
  - libtiff=4.0.6=vc14_2
  - matplotlib=1.5.1=np111py35_0
  - mkl=11.3.3=1
  - pyqt5=5.6
  - numpy=1.11.0=py35_1
  - pandas=0.18.1=np111py35_0
  - patsy=0.4.1=py35_0
  - pillow=3.2.0=py35_1
  - pip=8.1.2=py35_0
  - pyparsing=2.1.4=py35_0
  - python=3.5.1=5
  - python-dateutil=2.5.3=py35_0
  - pytz=2016.4=py35_0
  - reportlab=3.3.0=py35_0
  - scipy=0.17.1=np111py35_0
  - setuptools=19.2=py35_0
  - six=1.10.0=py35_0
  - statsmodels=0.6.1=np111py35_1
  - tk=8.5.18=vc14_0
  - vs2015_runtime=14.00.23026.0=0
  - wheel=0.29.0=py35_0
  - zlib=1.2.8=vc14_3
  - pip:
    - backports.shutil-get-terminal-size==1.0.0
    - future==0.15.2
    - pefile==2016.3.28
    - pyinstaller==3.2
    - pypiwin32==219
prefix: C:\Users\Andrew\Anaconda3\envs\ves_env_win64
```

**Comment by mmcauliffe** [comment]
So it should look like:

```
name: ves_env_win64
dependencies:
  - backports=1.0=py35_0
  - bzip2=1.0.6=vc14_3
  - cycler=0.10.0=py35_0
  - decorator=4.0.10=py35_0
  - freetype=2.5.5=vc14_1
  - get_terminal_size=1.0.0=py35_0
  - jpeg=8d=vc14_0
  - libpng=1.6.22=vc14_0
  - libtiff=4.0.6=vc14_2
  - matplotlib=1.5.1=np111py35_0
  - mkl=11.3.3=1
  - pyqt5=5.6
  - numpy=1.11.0=py35_1
  - pandas=0.18.1=np111py35_0
  - patsy=0.4.1=py35_0
  - pillow=3.2.0=py35_1
  - pip=8.1.2=py35_0
  - pyparsing=2.1.4=py35_0
  - python=3.5.1=5
  - python-dateutil=2.5.3=py35_0
  - pytz=2016.4=py35_0
  - reportlab=3.3.0=py35_0
  - scipy=0.17.1=np111py35_0
  - setuptools=19.2=py35_0
  - six=1.10.0=py35_0
  - statsmodels=0.6.1=np111py35_1
  - tk=8.5.18=vc14_0
  - vs2015_runtime=14.00.23026.0=0
  - wheel=0.29.0=py35_0
  - zlib=1.2.8=vc14_3
  - pip:
    - backports.shutil-get-terminal-size==1.0.0
    - future==0.15.2
    - pefile==2016.3.28
    - pyinstaller==3.2
    - pypiwin32==219
    - pyqt5==5.6
prefix: C:\Users\Andrew\Anaconda3\envs\ves_env_win64
```
Note the last line of the `-pip` section.  I'm assuming those are packages listed under the pip section get installed via `pip` instead of `conda`, since I'm pretty sure pyinstaller doesn't have a conda package.

**Comment by ajoros** [comment]
You're awesome! Got it working! Now to figure out some permission errors I'm having. Thanks again @mmcauliffe ! 

On a side note, after installing my Anaconda Cloud environment I only see `qt 4.8.7` installed while I remember your Anaconda Cloud pyqt5 package installed qt5. Why is PyQt5 working when I dont have qt5 installed like I did with your package?

**Comment by mmcauliffe** [comment]
The PyQt5 wheel that they built has the Qt5 libraries prebuilt inside, whereas the conda distribution would build them and have them available system wide for a pyqt5 package to link against.  Chances are, nothing besides pyqt5 will need to link against them, so it should be fine.

**Comment by ajoros** [comment]
Great, thanks for the assistance bud. Much appreciated. Are you going to be at SciPy?

**Comment by mmcauliffe** [comment]
Oh, no, there's a conference in my field that's happening at the same time.  Maybe next year!

**Comment by ajoros** [comment]
Bummer, oh well. Thanks for the assistance again!

On Thu, Jun 30, 2016 at 2:40 PM, Michael McAuliffe <notifications@github.com

**Comment by mmcauliffe** [comment]
Happy to help!
